### ISSUE TITLE
Create data folder and add data for notebooks

### ISSUE THREAD

**Issue by jekwatt** [BCM-HGSC/pycon2020_excel_poster#2]
Data used for notebooks:

https://github.com/seaborn/seaborn-data

Load built-in seaborn datasets:

```
import seaborn as sns
import pandas as pd
import matplotlib.pyplot as plt

iris = sns.load_dataset("iris")
```

**Comment by jekwatt** [comment]
Issue resolved in PR #10

**Comment by jekwatt** [comment]
Closing as this is done.
